src/txn/txn_rollback_to_stable.c

```diff
@@ -1210,12 +1298,30 @@ __rollback_to_stable_btree_apply(WT_SESSION_IMPL *session)
                     prepared_updates = true;
             }
             WT_ERR_NOTFOUND_OK(ret, false);
+            ret = __wt_config_subgets(session, &cval, "newest_txn", &value);
+            if (value.len != 0)
+                rollback_txnid = (uint64_t)value.val;
+            WT_ERR_NOTFOUND_OK(ret, false);
             ret = __wt_config_subgets(session, &cval, "addr", &value);
             if (ret == 0)
                 addr_size = value.len;
             WT_ERR_NOTFOUND_OK(ret, false);
         }
         max_durable_ts = WT_MAX(newest_start_durable_ts, newest_stop_durable_ts);
+        has_txn_updates_gt_than_ckpt_snap =
+          WT_CHECK_RECOVERY_FLAG_TXNID_CKPT_SNAPMIN(session, rollback_txnid);
+
+        if (F_ISSET(S2C(session), WT_CONN_RECOVERING))
+            __wt_verbose(session, WT_VERB_RTS,
```

Comment:
The verbose macro should be `WT_VERB_RECOVERY_RTS(session)` according to the latest code.

Reply:
Updated